roles/openshift_node/tasks/main.yml

```diff
@@ -59,6 +68,7 @@
     dest: /etc/sysconfig/{{ openshift.common.service_type }}-node
     regexp: "{{ item.regex }}"
     line: "{{ item.line }}"
+    create: yes
```

Comment:
Yeah it's put down by the rpm but we should be setting the loglevel however when looking at #642 it appears we're not doing something right as it's always using the default. I'd say drop it unless you find it's necessary.